src/ImageSharp/Advanced/AotCompilerTools.cs

```diff
@@ -99,5 +123,22 @@ private static void AotCompileDithering<TPixel>()
             TPixel pixel = default;
             test.Dither<TPixel>(new ImageFrame<TPixel>(Configuration.Default, 1, 1), pixel, pixel, 0, 0, 0, 0, 0, 0);
         }
+
+        /// <summary>
+        /// This method pre-seeds the decoder for a given pixel format in the AoT compiler for iOS.
+        /// </summary>
+        /// <param name="decoder">The image decoder to seed..</param>
+        /// <typeparam name="TPixel">The pixel format.</typeparam>
+        private static void AotDecoder<TPixel>(IImageDecoder decoder)
+            where TPixel : struct, IPixel<TPixel>
+        {
+            try
+            {
+                decoder.Decode<TPixel>(Configuration.Default, null);
+            }
+            catch
```

Comment:
This isn't nice, but can't think of a better solution.

Reply:
Yeah I agree.  Would be nice if there were a way to make a dummy call to `Decode` without generating exceptions.